walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -79,15 +94,15 @@
    */
   public Collection<TimeRange> queryHelper(Collection<Event> separateEvents, MeetingRequest request) {
```

Comment:
nit: queryHelper is a bit vague - it's not clear how the query is helped or what the help is. Can we rename to something more descriptive e.g. findAllViableTimeRanges or something along those lines

Reply:
Sure. Done.